src/modules/Common/managers/ServiceManager.ts

```diff
@@ -67,6 +78,10 @@ export default class ServiceManager {
     this.id = ServiceManager.deriveIdFromName(name);
     this.declarationFilePath = `declarations/${this.id}.json`;
     this.historyFilePath = `declarations/${this.id}.history.json`;
+    this.author = {
+      name: author?.name || 'Anonymous Contributor',
+      email: author?.email || 'anonymous@contribute.opentermsarchive.org',
```

Comment:
This duplicates the [default values in `ContributorForm.tsx`](https://github.com/OpenTermsArchive/contribution-tool/pull/135/files#diff-9d1c0d6f31651577385f1d5f8856df453321b1f830a10e0b0610e28961fddaf6R9) that are set with `DEFAULT_CONTRIBUTOR_NAME` and `DEFAULT_CONTRIBUTOR_EMAIL`.

Reply:
Right,
as this value is used on both frontend and server, i will have to add it to the `publicRuntime` next config.